wallet/wallet.go

```diff
@@ -5908,7 +5912,7 @@ func (w *Wallet) ForUnspentUnexpiredTickets(ctx context.Context,
 		int32(params.TicketExpiry+uint32(params.TicketMaturity)-requiredConfs)
 	startBlock := NewBlockIdentifierFromHeight(startBlockNum)
 	endBlock := NewBlockIdentifierFromHeight(blockHeight)
-	return w.GetTickets(ctx, iter, startBlock, endBlock)
+	return errors.Join(w.GetTickets(ctx, iter, startBlock, endBlock))
```

Comment:
not sure what this is trying to do.

Reply:
Good catch, I meant for that to be...

```suggestion
	return errors.Join(errs, w.GetTickets(ctx, iter, startBlock, endBlock))
```